app/controllers/api/v1/users_controller.rb

```diff
@@ -44,10 +44,6 @@ def update
     [].tap do |update_email_user_ids|
```

Comment:
Is this tap to support a multi-user updates? FYI, that yield-before-save bug in updatable_resource is still outstanding, I think. I'll get a one-line PR done up now.

I realize this was in the other PR that this is sort-of reverting, but I wanted to be clear.

Reply:
Yeah the tap is to collect all the updated user ids to use outside the transaction block.